src/pages/NewGroupPage.js

```diff
@@ -43,6 +43,10 @@ const propTypes = {
     ...windowDimensionsPropTypes,
 };
 
+const defaultProps = {
+    reports: {},
+};
+
```

Comment:
Whats the efficiency here?

Reply:
None :) 

Just fixing the `NewGroupPage` loading with `null` reports in some cases.

Reply:
Probably will undo a bit of this. Realizing that maybe these modals don't really need to be lazy loaded.

Reply:
Gonna fix this in another issue. Some kind of bug but unrelated to these changes.